meson.build

```diff
@@ -232,11 +232,16 @@ if get_option('enable-python')
   endif
 endif
 
-# nnfw ( details in https://review.tizen.org/gerrit/p/platform/core/ml/nnfw )
-if get_option('enable-nnfw')
-  add_project_arguments('-DENABLE_NNFW=1', language: ['c', 'cpp'])
-  # For tf-lite/nnapi, enable-nnfw allows to use nnfw as a backend.
-  # This also enables nnfw as a subplugin for Tizen.
+# nnfw-runtime ( details in https://review.tizen.org/gerrit/p/platform/core/ml/nnfw )
+if get_option('enable-nnfw-runtime')
+  add_project_arguments('-DENABLE_NNFW_RUNTIME=1', language: ['c', 'cpp'])
```

Comment:
This option, `ENABLE_NNFW_RUNTIME`, seems not used yet. Is it reserved for future usage? Then, I think it's better to mention it somewhere.

Reply:
It is to use Tizen /platform/core/ml/nnfw as yet another tensor-filter subplugin.

Reply:
> This option, `ENABLE_NNFW_RUNTIME`, seems not used yet. Is it reserved for future use? Then, I think it's better to mention it somewhere.

Adding this option creates confusion between using NNFW a tensor-filter subplugin, and NNFW as a backend for tensorflow-lite. Hence, this option is added here, and existing ENABLE_NNFW renamed, to lower burden on the other PR (#1856)